specification/reservations/resource-manager/Microsoft.Capacity/preview/2019-04-01/examples/CalculateReservationOrder.json

```diff
@@ -11,7 +11,7 @@
         "billingScopeId": "/subscriptions/ed3a1871-612d-abcd-a849-c2542a68be83",
         "term": "P1Y",
         "billingPlan": "Monthly",
-        "quantity": "1",
+        "quantity": 1,
```

Comment:
This was an existing example which had a syntax issue. Fixed it to pass model validation.